generator/projectmanager.h

```diff
@@ -77,5 +70,7 @@ struct ProjectManager {
     std::string includeRecovery(llvm::StringRef includeName, llvm::StringRef from);
 
 private:
+    std::vector<ProjectInfo> systemProjects();
```

Comment:
should it be static, if it does not touch the member.  (at least, should be const.)
Since it is called before the class is initialized, it is safer to make it static.

The alternative would be to call it initSystemProjects()